.gitignore

```diff
@@ -2,6 +2,9 @@
 # Those patterns should go in a global gitignore or repository-specific excludes.
 # See http://www.gwtproject.org/makinggwtbetter.html#global_gitignore
 
-target/
-
+*/target/*
```

Comment:
just plain `target/` should already exclude all target directories in all projects, right?

Reply:
done ... missed that one.